core/performance_test/peakflops/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+# Copied from unit test, could be DRYd
+
+# It only makes sense to run this if testing is enabled, as we depend on gtest
+if(NOT Cabana_ENABLE_TESTING)
+  return()
+endif()
+
+# we only want to run the below if we have access to "#pragma omp simd"
+include(CheckCXXSourceCompiles)
+check_cxx_source_compiles(
+      "int main(void) {
+      #pragma omp simd
+      return 0;
+      }" HAVE_OMP_SIMD)
+
+if (NOT HAVE_OMP_SIMD)
+    # No point running these tests without omp simd
+    message(WARNING "Compiler doesn't seem to support `#pragma simd`, skipping performance tests")
+    return()
+endif()
+
+set(GTEST_SOURCE_DIR ${CMAKE_SOURCE_DIR}/gtest)
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
+
+include_directories(${GTEST_SOURCE_DIR})
+include_directories(${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
I don't think this is needed.

Reply:
ill try removing (just `CMAKE_CURRENT_BINARY_DIR` right?)

Reply:
Yes.